package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@etherspot/transaction-kit",
   "description": "React Etherspot Transaction Kit",
-  "version": "0.6.12",
+  "version": "0.6.13",
```

Comment:
Minor point but this feels like a minor upgrade as we're introducing new hook variables?

Reply:
Updated to `0.7.0`